templates/bin/check

```diff
@@ -60,6 +60,7 @@ if [ ! $? -eq 0 ]; then
   exit 1
 fi
 
+csscomb app/assets/stylesheets/**/*.scss
```

Comment:
@nicolaseckhart @rbuetzer this doesn't work like that, since the changes will be unstaged / not committed. Suggested workaround:
```git status``` before the command, ```git status``` after the command, and if it has changed, exit(1)